contacts/constants.js

```diff
@@ -0,0 +1,141 @@
+// MODULE FOR CONSTANTS
+
+const DELETE_ALL_BUTTON = 'delete-all-button';
+const CLEAR_ALL_BUTTON = 'clear-button';
+const FAVS_LIST = 'favs-list';
+const CONTACTS_LIST = 'contacts-list';
+const ADD_BUTTON = 'add-button';
+const FIRST_NAME = 'first-name';
+const LAST_NAME = 'last-name';
+const IMAGE_PROFILE = 'profile-image';
+const ADD_FORM_MAIN = 'add-form';
+
+export const DOCUMENT_ELEMENTS = {
+  FAVOURITES: {
+    FAVS_LIST,
+    CLEAR_ALL_BUTTON
+  },
+  CONTACTS: {
+    CONTACTS_LIST,
+    DELETE_ALL_BUTTON
+  },
+  ADD_FORM: {
+    ADD_FORM_MAIN,
+    ADD_BUTTON,
+    FIRST_NAME,
+    LAST_NAME,
+    IMAGE_PROFILE
+  }
+};
+
+// CONSTANTS FOR ID
+
+const ID_LENGTH = 19;
+
+export const ID_CONSTANTS = {
+  ID_LENGTH
+};
+
+// TOP N ELEMENTS
+
+export const NUMBER_TOP = 5;
+
+// PREFIXES FOR CONTACTS IDs CONVENTION
+
+export const PREFIXES_ID_CONTACTS = {
+  LIST_ITEM: 'li-',
+  PLUS_BUTTON: 'plus-button-',
+  MINUS_BUTTON: 'minus-button-',
+  COUNTER: 'count-',
+  IMAGE: 'image-',
+  CHECKBOX: 'checkbox-',
+  FULL_NAME: 'full-name-',
+  TRASH: 'trash-'
+};
+
+// NUMBERS
+
+export const NINE = 9;
```

Comment:
there's no point in having a number into a constant if it doesn't mean something.
if it means something, please rename it to what it means, like you did for MAX_NUMBER_OF_LIKES

Reply:
snd on top of them, group them with a suggestive name